src/Plugin/smart_content/Condition/SmartCDNCondition.php

```diff
@@ -42,13 +42,43 @@ public function getAttachedSettings() {
     // Determine CDN value based on derivative id.
     $cdn_value = NULL;
     switch ($derivative_id) {
-      case 'geo':
```

Comment:
@kmjacobs Is this a breaking change? Or should we stick this as a Major update to the module?

Reply:
Confirmed a breaking change. But we're good to do a major release, along with having release notes.